api/plugins/action/deploy_bulk.py

```diff
@@ -44,8 +53,10 @@ def deploy_bulk(client: GqlClient, build_vars: list, name: str, envs: list) -> d
             "envs": envs,
         }
     )
-
     display.v(f"GraphQL query result: {res}")
+
+    if 'errors' in res:
+        raise AnsibleError("Unable to create bulk deployment.", res['errors'])
```

Comment:
@steveworley I added this so that we get more helpful errors quickly. With the try..except, there was no way to find out what the error was.